index.html

```diff
@@ -0,0 +1,186 @@
+  <link rel="stylesheet" href="style.css">
+  <title>Middle-Version-Mobile Setup</title>
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;800&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <header class="head"> 
+      <nav class="nav-bar"> 
+          <a class="logo" href="#"><h3>My Logo<h3></a>
+          <img class="union" src="./images/Union.png" alt="Hamburger menu">
+      </nav>  
+    </header>
+    <section class="headline">  
+         <h1>I'm Shadrach Glad to see you!</h1>
+
+         <p>I'm a software developer! I can help you build a product feature or website. Look
+           through some of my work and experience! If you like what you see and<br>have a project 
+            you need coded, don't hesitate to contact me.
+         </p> 
+        <a href="#" class="let-connect">LET'S CONNECT </a>
+      <ul class="icons-social">
+         <li><a class="icon icon-twitter" href="Twitter"><img src="./icon-social/Twitter.png" alt="Twitter icon"></a></li>
+         <li><a class="icon icon-linkedin" href="Linkedin"><img src="./icon-social/Linkedin.png" alt="Linkedin icon"></a></li>
+         <li><a class="icon icon-medium" href="Messenger"><img src="./icon-social/Messenger.png" alt="Messenger icon"/></a></li>
+         <li><a class="icon icon-github" href="Github"><img src="./icon-social/Github.png" alt="Github icon"/><a></li>
+         <li><a class="icon icon-iOS" href="iOS"><img src="./icon-social/iOS.png" alt="iOS icon"></a></li>
+      </ul>
+    </section> 
+    <section class="work-container">
+        <div class="work-items">
+            <a class="design" href="#"><img src="./images/Designer.png" alt="Designer"></a>
+            <h1 class="work-title-1">Tonic</h1>
+            <ul class="projects">
+                <li class="work-name"><a href="#">CANOPY</a></li>
+                <li class="counter"><img src="./images/Counter.png"></li>
+                <li><a class="backend-dev">Back End Dev</a></li>
+                <li class="counter"><img src="./images/Counter.png"></li>
+                <li><a class="backend-dev" href="#">2015</a></li>
+            </ul>
+            <p class="content-work"> 
+                A daily selection of privately personalized reads; no accounts or sign-ups required.  
+            </p>
+            <ul class="content-laguages">
+               <li class="lang"><a href="#">html</a></li>
+               <li class="lang"><a href="#">css</a></li>
+               <li class="lang"><a href="#">javascript</a></li>
+            </ul>
+            <div>
+                <button type="submit" class="project-btn">See Project</button>
+            </div>
+        </div>           
+        <div class="work-items">
+           <a class="design" href="#"> <img src="./images/Avialability.png" alt="Availability"></a>
+             <h1 class="work-title">Multi-Post 
```

Comment:
:red_circle:  Please make sure you change this header tag to any other header tag (h2,h3 ...); As requested by the previous reviewer your HTML should not have more than one  `<h1> ` tag.  H1 on the page indicates what the entire web page is about. And the rest of the headings follow a hierarchical structure, with H2 for each topic section and any subsections to those being indicated with H3 and if the subtopic has subtopics then going down as low as H4.

I think this code should be `<h3 class="work-title">Multi-Post <br>Stories </h3>`